src/screens/WalletConnect/WalletConnectCallRequest.js

```diff
@@ -316,13 +359,14 @@ class WalletConnectCallRequestScreen extends React.Component<Props, State> {
 }
 
 const mapStateToProps = ({
-  assets: { supportedAssets },
+  assets: { supportedAssets, balances },
   contacts: { data: contacts },
   session: { data: session },
 }) => ({
   contacts,
   session,
   supportedAssets,
+  balances,
```

Comment:
When we merge this into smart-wallet branch, we need to use a selector to get the balance

Reply:
👍